input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "signal-profile-us-core-service-request",
+  "url": "https://signalbhn.org/fhir/StructureDefinition/StructureDefinition-profile-SignalUSCoreServiceRequest",
+  "name": "StructureDefinition-profile-SignalUSCoreServiceRequest",
```

Comment:
Need to change this:  
```
Message  : Improper name: 'StructureDefinition-profile-SignalUSCoreServiceRequest'. Name should be usable as an identifier for the module by machine processing applications such as code generation (capital letter followed by any number of letters, digits or underscores with a maximum of 255 characters).
Project  : Signal MSO FHIR Profiles
Resource : StructureDefinition-profile-SignalUSCoreServiceRequest
Id       : sdf-0
Severity : Warning
Source   : StructureDefinition-profile-SignalUSCoreServiceRequest
Context  : name
```

Reply:
@enjoysparkling I have made the requested changes for id, name and URL elements.

#e8d56dabc9e357bbbeb04bff77e39bc121acc4c7

Reply:
Do we need to do the same for Signal US Core Organization profile and Signal Health Care Service profiles?
@enjoysparkling